tests/api/v1/samples/post.js

```diff
@@ -254,19 +253,35 @@ describe('tests/api/v1/samples/post.js >', () => {
     });
   });
 
-  describe('aspect isPublished false >', () => {
+  describe('subject isPublished false >', () => {
     let sampleToPost;
-    let token;
-
-    before((done) => {
-      tu.createToken()
-      .then((returnedToken) => {
-        token = returnedToken;
+    beforeEach((done) => {
```

Comment:
is the "beforeEach" hook required? just a "before" hook will do.

Reply:
see comment below